src/App.js

```diff
@@ -1,26 +1,88 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {BrowserRouter as Router, Link, Route, Switch} from "react-router-dom";
 
 function App() {
   return (
```

Comment:
your return should look something like this: 
return (
 <RoutesComponent />
 <LinksComponent />
);

Reply:
la ce te referi mai exact, nu asta am facut?